modules/ROOT/pages/csrf-security.adoc

```diff
@@ -46,7 +46,7 @@ The `X-Bonita-API-Token` is valid for the whole client session.
 Disabling the CSRF protection is not recommended as it will makes Bonita REST API unprotected from malicious attacks.
 
 But if you need to, the deactivation of the CSRF protection can be done in the file `security-config.properties` for the whole platform.
-The default version of this file is located in `setup/platform_conf/initial/platform_portal`. In order to change the configuration on an installation whose platform has already been initialized, use the xref:bonita-bpm-platform-setup.adoc[platform setup tool] to retrieve the current configuration and update the file in `setup/platform_conf/current/platform_portal`. Then use the tool again to save your changes into to the database.
+The default version of this file is located in `setup/platform_conf/initial/platform_portal`. In order to change the configuration on an installation whose platform has already been initialized, use the xref:bonita-platform-setup.adoc[platform setup tool] to retrieve the current configuration and update the file in `setup/platform_conf/current/platform_portal`. Then use the tool again to save your changes into to the database.
```

Comment:
setup/platform_conf/initial/platform_portal` => shouldnt we change the name? there is portal in it.

Reply:
This folder is used for tenant configuration, so it remains. Honestly, not the first place where it's referenced and if needed, it can be changed.